### PR TITLE
add code to pull emulated-results and static-results

### PR DESCRIPTION
**Pull request by CoffeeExpress**

### CODE REVIEW

File: centrifuge_cli/main.py [hunk 1]
```diff
@@ -201,23 +223,35 @@ def code_summary(cli):
 
 
 @report.command(name='code-static')
```
Comment (eacmen) [inline centrifuge_cli/main.py]: you realize this is not needed right? The underscore gets replaced with hyphen in the CLI Interface.

File: centrifuge_cli/main.py [hunk 2]
```diff
@@ -201,23 +223,35 @@ def code_summary(cli):
...
-@click.option('--exid', required=True, metavar='EXID', help='Extraction ID from code-summary output')
-@click.option('--path', required=True, metavar='PATH', help='File path that you want to get analysis results for')
+@click.option('--exid', default=0, metavar='EXID', help='Extraction ID from code-summary output')
```
Comment (eacmen) [inline centrifuge_cli/main.py]: these defaults make no sense you get no results

File: centrifuge_cli/main.py [hunk 3]
```diff
@@ -54,11 +52,35 @@ def build_url(self, path, query_list):
         query = '&'.join(default_query)
         return urlunparse((self.endpoint_scheme, self.endpoint_netloc, path, None, query, None))
 
-    def do_GET(self, path, query_list=None):
-        url = self.build_url(path, query_list)
+    def do_GET(self, path, query_list=None, get_all=False):
 
-        res = requests.get(url)
-        res.raise_for_status()
+        # handle paginated queries
```
Comment (eacmen) [inline centrifuge_cli/main.py]: I don't understand why pagination is required when the --limit parameter is supported.

Comment (CoffeeExpress) [reply centrifuge_cli/main.py]: There are several api endpoints that have a max limit of 100. You have to paginate.

File: centrifuge_cli/main.py (hunk 2 above)
Comment (CoffeeExpress) [reply centrifuge_cli/main.py]: They don't make any sense, but you get all results with exid set to zero and no path argument.

File: centrifuge_cli/main.py (hunk 1 above)
Comment (CoffeeExpress) [reply centrifuge_cli/main.py]: Nope, didn't know that

Comment (CoffeeExpress) [reply centrifuge_cli/main.py]: Also, that's your code.

Comment (eacmen) [reply centrifuge_cli/main.py]: not sure why github showed that line as added.

File: centrifuge_cli/main.py [hunk 4]
```diff
@@ -267,7 +294,7 @@ def users(cli):
 @users.command(name="list")
 @pass_cli
 def user_list(cli):
-    click.echo(cli.do_GET('/api/user'))
+    click.echo(cli.do_GET('/api/user', pagination=100))
```
Comment (eacmen) [inline centrifuge_cli/main.py]: do_GET does not take pagination parameter

**Pull request merged**